docs/notes.rst

```diff
@@ -0,0 +1,42 @@
+notes
+=====
+
+.. dfhack-tool::
+    :summary: Manage map-specific notes
+    :tags: fort interface map
+
+The `notes` tool enables players to annotate specific tiles
+on the Dwarf Fortress game map with customizable notes.
+
+Each note is displayed as a green pin on the map and includes a one-line title and a detailed comment.
+
+It can be used to e.g.:
+ - marking plans for future constructions
+ - explaining mechanisms or traps
+ - noting historical events
+
+Usage
+-----
+
+::
+
+    notes add
+
+Add new note in the current position of the keyboard cursor.
+
+Creating a Note
+~~~~~~~~~~~~~~~
```

Comment:
to keep in line with documentation for existing tools, these can be dashes -- making the sections peers of Usage

Reply:
In my understanding, this sections are "Usage" explanations. Or should/can be grouped on some other section, as they all describe how to use the too.

Reply:
I think it's fine to leave them all as top-level sections. Nothing else has subheaders under Usage